README.md

```diff
@@ -6,13 +6,74 @@ Are you looking for maximum flexibility and efficiency in your next Python libra
 
 👉 Introducing **draive** - an open-source Python library under the Miquido AI Kickstarter framework, designed to simplify and accelerate the development of LLM-based applications. Get started with draive to streamline your workflow and build powerful, efficient apps with ease.
 
-## Examples repository
+## 🚀 Quick start
 
-Fully functional examples of using the Draive library are available in a separate repository
+You can jump into the code right away and learn how to use draive using our interactive [guides](https://github.com/miquido/draive/tree/main/guides) and check out our [cookbooks](https://github.com/miquido/draive/tree/main/cookbooks) for simple solutions of common problems.
```

Comment:
Dive straight into the code and learn how to use draive with our interactive [guides](https://github.com/miquido/draive/tree/main/guides). For quick solutions to common problems, explore our [cookbooks].

I think it will be more clear this way.